README.md

```diff
@@ -73,7 +76,7 @@ Special builds can be created that exclude subsets of jQuery functionality.
 This allows for smaller custom builds when the builder is certain that those parts of jQuery are not being used.
 For example, an app that only used JSONP for `$.ajax()` and did not need to calculate offsets or positions of elements could exclude the offset and ajax/xhr modules.
 
-Any module may be excluded except for `core`, and `selector`. To exclude a module, pass its path relative to the `src` folder (without the `.js` extension).
+Any module may be excluded except for the `core`, and `selector` modules. To exclude a module, pass its path relative to the `src` folder (without the `.js` extension).
```

Comment:
`core` is the name here, why do we need the article? We're not writing "the Firefox". "except for" also suggests what's mentioned are modules.